sim.py

```diff
@@ -87,6 +141,8 @@ def main():
         '--talents', help='indicate talent build for output.', choices=config["builds"].keys())
     parser.add_argument(
         '--covenant', help='indicate covenant build for output.', choices=config["covenants"])
+    parser.add_argument(
+        '--local', help='indicate covenant build for output.', action='store_true')
```

Comment:
should change help text here

Reply:
Should be fixed, damn copied it over from the covenant and forgot to change it.